testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/microprofile/restclient/RestClientProxyTest.java

```diff
@@ -28,6 +29,7 @@
 import io.reactivex.Single;
 
 @RunWith(Arquillian.class)
+@RunAsClient
```

Comment:
Why is this needed? I mean, it's not a problem, just wondering why.

Reply:
It is needed when security manager is being used.  Without it arquillian throws exception,
java.util.NoSuchElementException